### PR TITLE
Scientific package tests on Athena.

### LINKED ISSUE #1176: Athena support
Similar support for Athena as in Soda SQL https://docs.soda.io/soda/warehouse_types.html#amazon-athena

### PR DESCRIPTION
**Pull request by m1n0**
Fix #1176

### CODE REVIEW

**Review by bastienboutonnet — COMMENTED** [review]
Nice fixes @m1n0 

Regarding the failure on the histograms calculation, I'm not sure what is going on yet. Do you have a way to know if the query is correctly executed?

I can't try to debug it now, because I don't have credentials but my hunches are:
- either something is wrong with the SQL or something with casing in the histogram query.

Let me know if you want me to check a few things and in that case I might want to see about credentials then.

File: soda/athena/soda/data_sources/athena_data_source.py [hunk 1]
```diff
@@ -118,3 +129,14 @@ def sql_get_table_names_with_count(
...
         return ""
+
+    def profiling_sql_cte_value_frequencies(self, table_name: str, column_name: str) -> str:
+        """Same as the inherited method, only difference is that Athena wants original column name in the GROUP BY clause instead of the alias name."""
+        column_name = self.quote_column(column_name)
+        return dedent(
+            f"""
+                SELECT {column_name} as value_name, count(*) as frequency
+                FROM {table_name}
+                GROUP BY {column_name}
```
[the remark this answers is not in the document]
Comment (m1n0) [reply soda/athena/soda/data_sources/athena_data_source.py:140]: indeed I didnt think of that, I like that solution better! changed

**Comment by bastienboutonnet** [comment]
@m1n0 I'll trust you on the SQL errors since I do not have a way to actually try an Athena db at the moment.

As far as I can tell, I think I like what you're introducing as it will standardise things very nicely, however, it looks like a lot of tests are failing as a result (as I'm sure you're aware). Let me know if there's anything I can help with further.

**Comment by m1n0** [comment]
I left one test skipped for athena, will deal with that one separately as some fixes here will help with other PRs.

**Pull request merged**